tests/fsstress/fsstress_test.go

```diff
@@ -0,0 +1,346 @@
+
+	t.Log("Add app to list")
+
+	edgeNode.SetApplicationInstanceConfig(append(edgeNode.GetApplicationInstances(), appInstanceConfig.Uuidandversion.Uuid))
+
+	tc.ConfigSync(edgeNode)
+
+	t.Log("Add processing of app running messages")
+
+	tc.AddProcInfo(edgeNode, checkAppRunning(appName))
+
+	appID, err := uuid.FromString(appInstanceConfig.Uuidandversion.Uuid)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	callback := func() {
+		fmt.Printf("--- app %s logs ---\n", appInstanceConfig.Displayname)
+		if err = tc.GetController().LogAppsChecker(edgeNode.GetID(), appID, nil, eapps.HandleFactory(eapps.LogJSON, false), eapps.LogExist, 0); err != nil {
+			t.Fatalf("LogAppsChecker: %s", err)
+		}
+		fmt.Println("------")
+	}
+
+	tc.WaitForProcWithErrorCallback(int(timewait.Seconds()), callback)
+}
+
+func getAppInstanceConfig(edgeNode *device.Ctx, appName string) *config.AppInstanceConfig {
+
+	var appInstanceConfig *config.AppInstanceConfig
+
+	for _, id := range edgeNode.GetApplicationInstances() {
+		appConfig, _ := tc.GetController().GetApplicationInstanceConfig(id)
+		if appConfig.Displayname == appName {
+			appInstanceConfig = appConfig
+			break
+		}
+	}
+	return appInstanceConfig
+}
+
+
+//TestAccess checks if SSH of app is accessible
+func TestAccess(t *testing.T) {
+
+	edgeNode := tc.GetEdgeNode(tc.WithTest(t))
+
+	setAppName()
+
+	appInstanceConfig := getAppInstanceConfig(edgeNode, appName)
+
+	if appInstanceConfig == nil {
+		t.Fatalf("No app found with name %s", appName)
+	}
+
+	appID, err := uuid.FromString(appInstanceConfig.Uuidandversion.Uuid)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	t.Log(utils.AddTimestamp("Add function to obtain EVE IP"))
+
+	tc.AddProcTimer(edgeNode, getEVEIP(edgeNode))
+
+	t.Log(utils.AddTimestamp("Add trying to access SSH of app"))
+
+	tc.AddProcTimer(edgeNode, projects.SendCommandSSH(&externalIP, sshPort, "ubuntu", "passw0rd", "exit", true, defaults.DefaultRepeatTimeout))
+
+	tc.ExpandOnSuccess(int(expand.Seconds()))
+
+	callback := func() {
+		fmt.Printf("--- app %s logs ---\n", appInstanceConfig.Displayname)
+		if err = tc.GetController().LogAppsChecker(edgeNode.GetID(), appID, nil, eapps.HandleFactory(eapps.LogJSON, false), eapps.LogExist, 0); err != nil {
+			t.Fatalf("LogAppsChecker: %s", err)
+		}
+		fmt.Println("------")
+	}
+
+	tc.WaitForProcWithErrorCallback(int(timewait.Seconds()), callback)
+
+}
+
+//TestRunStress run fsstress test on guest vm
+func TestRunStress(t *testing.T) {
+	edgeNode := tc.GetEdgeNode(tc.WithTest(t))
+	setAppName()
+
+	appInstanceConfig := getAppInstanceConfig(edgeNode, appName)
+
+	if appInstanceConfig == nil {
+		t.Fatalf("No app found with name %s", appName)
+	}
+
+	appID, err := uuid.FromString(appInstanceConfig.Uuidandversion.Uuid)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	ex, err := os.Executable()
+	if err != nil {
+			panic(fmt.Sprintf("AAAA %v",err))
+	}
+
+	dir := filepath.Dir(ex)
+	patch := filepath.Join(dir, "/../../", "tests/fsstress/testdata/run-script.sh")
```

Comment:
Can you please use eden.tests option from current config here to build path? We can run tests from another place or we can put our executable elsewhere.

Reply:
I added the -script_path parameter

Reply:
And I added eden.tests option from the current config here to build a path